### ISSUE TITLE
After upgrading from version 1 to version 2....

### ISSUE THREAD

**Issue by mdius** [concretecms-community-store/community_store#636]
Been getting help from Nour with a few issues, and decided to upgrade while he was helping to help with any possible issues. Found 1 major issue later, after upgrading my main site.
On certain pages it would list a product from a different page in the product list.
https://www.mytinytown.com/liquid-level-controls/mechanical-floats/heavy-duty-float-switch/normally-open-pump-down
This is correct info.

https://www.mytinytown.com/liquid-level-controls/specialty-float-switches/high-amp-float-switch/normally-open-pump-down
This is wrong info.

I verified the links point to the correct page, which never changed, in the dashboard. I went through them and repointed them with no luck.
I deleted the block and re-added it, no luck.
What I did to fix it was, changed it from Categorize under current page to Categorize under a specific page. Which is actually the same page.

If you need login info let me know, it has changed, maybe.

**Comment by Mesuva** [comment]
?

**Comment by mdius** [comment]
Sorry, my finger hit enter while doing the subject title. So it added it before I could put content.

**Comment by Mesuva** [comment]
I'm not sure what would be causing this, but it's worth clearing caches and doing a doctrine entity refresh.

**Comment by mdius** [comment]
I did the cache, but didn't try the doctrine. The test site [mytinytown] has all cache off and doctrine development mode on.
But after clearing cache and refreshing entities, no difference.
I even deleted the files/tmp and files/cache folders, no help. I'm sure it wasn't a good idea, but it's my test site and worth a try.

If I add the product to a different page it will show up, so I suppose if I deleted the page the products are attached to, it would work.
It's almost like the core of Concrete is causing the issue?
